src/components/CompareResults/ResultsMain.tsx

```diff
@@ -63,46 +63,49 @@ function ResultsMain() {
 
   return (
     <Container className={styles.container} data-testid='results-main'>
+      <header>
+        <div className={styles.title}>Results</div>
+        <Grid container className={styles.content} spacing={2}>
+          <Grid item md={6} xs={12}>
+            <SearchInput onChange={setSearchTerm} />
+          </Grid>
+          <Grid item xs>
+            <FormControl sx={{ width: '100%' }}>
+              <FrameworkDropdown
+                frameworkId={frameworkIdVal}
+                size='small'
+                variant='outlined'
+                onChange={onFrameworkChange}
+              />
+            </FormControl>
+          </Grid>
+          <Grid item xs>
+            <RevisionSelect />
+          </Grid>
+          <Grid item xs>
+            <DownloadButton resultsPromise={results} />
+          </Grid>
+        </Grid>
+      </header>
+      {/* Using a key in Suspense makes it that it displays the fallback more
+          consistently.
+          See https://github.com/mozilla/perfcompare/pull/702#discussion_r1705274740
+          for more explanation (and questioning) about this issue. */}
       <Suspense
         fallback={
-          <Box display='flex' justifyContent='center'>
+          <Box display='flex' justifyContent='center' sx={{ marginTop: 3 }}>
```

Comment:
@Carla-Moz I bumped this to 3 instead of the previous 2, so that it's the same as the table margin.